prize-tool/src/main.rs

```diff
@@ -0,0 +1,124 @@
+mod availability;
+mod confirmation_latency;
+mod prize;
+mod rewards_earned;
+mod utils;
+
+use clap::{crate_description, crate_name, crate_version, value_t, value_t_or_exit, App, Arg};
+use solana_core::blocktree::Blocktree;
+use solana_core::blocktree_processor::{process_blocktree, ProcessOptions};
+use solana_sdk::genesis_block::GenesisBlock;
+use solana_sdk::native_token::sol_to_lamports;
+use solana_sdk::pubkey::Pubkey;
+use std::path::PathBuf;
+use std::process::exit;
+use std::str::FromStr;
+
+fn main() {
+    solana_logger::setup();
+
+    let matches = App::new(crate_name!())
+        .about(crate_description!())
+        .version(crate_version!())
+        .arg(
+            Arg::with_name("ledger")
+                .short("l")
+                .long("ledger")
+                .value_name("DIR")
+                .takes_value(true)
+                .required(true)
+                .help("Use directory for ledger location"),
+        )
+        .arg(
+            Arg::with_name("starting_balance")
+                .long("starting_balance")
+                .value_name("SOL")
+                .takes_value(true)
+                .default_value("1000")
+                .help("Starting balance of validators at the beginning of TdS"),
+        )
+        .arg(
+            Arg::with_name("solana_baseline")
```

Comment:
solana_baseline -> baseline_validator perhaps?  Baseline might not be a "solana" validator